plutus-core/plutus-ir/test/PlutusIR/Transform/Rename/Tests.hs

```diff
@@ -13,18 +13,26 @@ import PlutusIR.Transform.Rename ()
 import Test.Tasty.QuickCheck
 
 test_rename :: TestTree
-test_rename = runTestNestedIn ["plutus-ir", "test", "PlutusIR", "Transform"] $
+test_rename =
+  runTestNestedIn ["plutus-ir", "test", "PlutusIR", "Transform"] $
     testNested "Rename" $
-        map
-            (goldenPir
-             (PLC.AttachPrettyConfig debugConfig . runQuote . runTestPass (const renamePass)) pTerm)
-            [ "allShadowedDataNonRec"
-            , "allShadowedDataRec"
-            , "paramShadowedDataNonRec"
-            , "paramShadowedDataRec"
-            ]
+      runGoldenPir <$>
+        [ "allShadowedDataNonRec"
+        , "allShadowedDataRec"
+        , "paramShadowedDataNonRec"
+        , "paramShadowedDataRec"
+        ]
   where
-    debugConfig = PLC.PrettyConfigClassic PLC.debugPrettyConfigName False
+    runGoldenPir =
+      goldenPir
+        ( PLC.AttachPrettyConfig prettyConfig
```

Comment:
I think we probably don't need that here, it was needed to adjust the default and now we want the default behavior. Anyways, not important.